default.nix

```diff
@@ -0,0 +1,69 @@
+let
+  # Pin some fairly new nixpkgs
+  sources = builtins.fetchTarball {
+    name = "nixpkgs-unstable-2021-07-06";
+    url = "https://github.com/nixos/nixpkgs/archive/291b3ff5af268eb7a656bb11c73f2fe535ea2170.tar.gz";
+    sha256 = "1z2l7q4cmiaqb99cd8yfisdr1n6xbwcczr9020ss47y2z1cn1x7x";
+  };
+
+  # For bootstrapping
+  pkgs = import sources {
+    overlays = [
+      (pkgs: super: {
+        # TODO replace with proper packaging once https://github.com/NixOS/nixpkgs/pull/128889 is merged
+        mold = pkgs.stdenv.mkDerivation {
+          pname = "mold";
+          version = "0.9.1";
```

Comment:
git head is usually unversioned except just after I make a new release. I'd remove `version` from this config file and just use `name = "mold"`.